libsql-server/src/http/admin/mod.rs

```diff
@@ -219,13 +221,19 @@ struct HttpDatabaseConfig {
     max_db_size: Option<bytesize::ByteSize>,
     #[serde(default)]
     heartbeat_url: Option<String>,
+    #[serde(default)]
+    jwt_key: Option<String>,
 }
 
 async fn handle_post_config<M: MakeNamespace, C>(
     State(app_state): State<Arc<AppState<M, C>>>,
     Path(namespace): Path<String>,
     Json(req): Json<HttpDatabaseConfig>,
 ) -> crate::Result<()> {
+    if let Some(jwt_key) = req.jwt_key.as_deref() {
+        // Check that the jwt key is correct
+        parse_jwt_key(jwt_key).context("Could not parse JWT decoding key")?;
```

Comment:
Would you have an example of something similar that's already in the codebase, please? @LucioFranco

Reply:
In this case, it looks like we have a lot of detailed error messages here https://github.com/tursodatabase/libsql/blob/4db7a4ce3dad009b98c34861a33e5785d896e4f7/libsql-server/src/auth.rs#L327 so I would just remove the call to `context` here.

In general, the pattern inside `parse_jwt_key` we should completely move away from because it tends to hide a lot of the actual errors that are useful for debugging due to the way anyhow uses debug over display. Happy to explain this more on slack but essentially we should avoid using `context` and instead when we want more detailed contextual errors use a new error enum that has those encoded into the variant via `thiserror` macro annotations.